pkgs/tools/security/ripasso/cursive.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib, rustPlatform, fetchFromGitHub, pkgconfig, ncurses, python3, openssl, libgpgerror, gpgme, xorg }:
+
+with rustPlatform;
+buildRustPackage rec {
+
+  version = "b64b1d9b697b2246b96db24a9d9ae571eab839aa";
+  name = "ripasso-cursive-${version}";
+
+  src = fetchFromGitHub {
+    owner = "cortex";
+    repo = "ripasso";
+    sha256 = "0vc56vjlq3xspvk77d4k49z9l6hs5707har7hgij0bs95k7w9lq0";
+    rev = version;
+  };
+
+  cargoSha256 = "0rsimk36v17fgkd3x9cln8z9hm1qslblzrxfssz6hfqxfm92k73p";
+  cargoBuildFlags = [ "-p ripasso-cursive" ];
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [
+    ncurses python3 openssl libgpgerror gpgme xorg.libxcb
+  ];
+  meta = with stdenv.lib; {
+    description = "A simple password manager written in Rust";
+    homepage = https://github.com/cortex/ripasso;
```

Comment:
We have an open RFC for deprecating the URL syntax that is likely to pass, so this should be a string.